src/main/java/org/agilemethod/pair10/grade_system/UI.java

```diff
@@ -1,30 +1,79 @@
 package org.agilemethod.pair10.grade_system;
 
+/**
+ *
+ * <h1>Main program</h1>
+ * The UI program implements an application that
+ * take the user request
+ * and print out to the screen
+ *
+ *
+ *
+ * @version 1.0
+ * @since 2018-05-07
+ */
+
 public class UI {
+    /**
+     * Construct a UI
+     */
     public UI() {
 
     }
 
+    /**
+     * continue prompt user ID until quit.
+     *
+     * @throws NoSuchIDException if there's no such id exist
+     * @throws NoSuchCommandException if there's no such command
+     */
     public void run() throws NoSuchIDException, NoSuchCommandException {
 
     }
 
+    /**
+     * This method ask user to insert the id.
+     *
+     * @return string of ID
+     */
     public String promptID() {
         return "";
     }
 
+    /**
+     * This method would check whether the input id is exist.
+     * @param id user input id
+     * @return true or false
+     * @throws NoSuchIDException if there's no ID have found
+     */
     public boolean checkID(String id) throws NoSuchIDException {
         return true;
     }
 
+    /**
+     * This method show welcome message of user
+     * @param id user input id
+     */
     public void showWelcomeMsg(String id) {
 
     }
 
+    /**
+     * This method give the four instruction on screen
```

Comment:
- remove `This method`
- `Prints` instead of `give`
- `commands` instead of `instruction`
- remove `on screen`